docs/root/faq/zone_aware_routing.rst

```diff
@@ -20,18 +20,23 @@ These are the requirements:
 .. code-block:: json
 
   {
-    "sds": "{...}",
-    "local_cluster_name": "cluster_a",
-    "clusters": [
-      {
-        "name": "cluster_a",
-        "type": "sds",
-      },
-      {
-        "name": "cluster_b",
-        "type": "sds"
-      }
-    ]
+    "cluster_manager": {
```

Comment:
What do you think about switching these to YAML? This is the suggested human readable format for v2 configs.

Reply:
Great, I'll change the format.